static/js/biomedical/disease/page.tsx

```diff
@@ -150,6 +159,21 @@ export class Page extends React.Component<PagePropType, PageStateType> {
           </>
         )}
         <br></br>
+        {!_.isEmpty(this.state.diseaseParent) && (
+          <>
+            <h5>Disease Ontology Hierachy</h5>
+            <p>
+              The hierarchy of diseases fetched from the Disease Ontology
+              database. The smallest sized and lightest colored circle
+              represents the child node or the disease of interest. As one
+              proceeds from left to right, the circle size keeps increasing and
+              circle color darkens, representing the subsequent nodes being
+              parents to the preceeding ones.
```

Comment:
update this description to match the new chart?

Reply:
Done!